src/system/SystemPacketBuffer.cpp

```diff
@@ -524,6 +524,7 @@ PacketBufferHandle PacketBufferHandle::NewWithData(const void * aData, size_t aD
     PacketBufferHandle buffer = New(aDataSize + aAdditionalSize, aReservedSize);
     if (buffer.mBuffer != nullptr)
     {
+        ChipLogError(chipSystemLayer, "yujuan:PacketBuffer: memcpy.");
```

Comment:
Please take this out or make it more meaningful?

Reply:
oops, accident.